cmd/openfga/openfga_test.go

```diff
@@ -0,0 +1,919 @@
+//go:build functional
+// +build functional
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	"github.com/openfga/openfga/pkg/id"
+	"github.com/openfga/openfga/pkg/testutils"
+	"github.com/stretchr/testify/require"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc"
+	grpcbackoff "google.golang.org/grpc/backoff"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	healthv1pb "google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+type OpenFGATester interface {
+	GetGRPCPort() string
+	GetHTTPPort() string
+	Cleanup() func()
+}
+
+type serverHandle struct {
+	grpcPort string
+	httpPort string
+	cleanup  func()
+}
+
+func (s *serverHandle) GetGRPCPort() string {
```

Comment:
Do we need this function? Or can we just make `grpcPort` in `serverHandle` public?

Reply:
Sounds good to me. Thanks!